configure.ac

```diff
@@ -153,6 +167,15 @@ AS_IF([test "x$WANTS_IPV6" = "xyes"], [
   USES_IPV6=yes
 ])
 
+AC_ARG_ENABLE([braille],
+  [AS_HELP_STRING([--disable-braille], [Do not enable barille character graphs])],
+  [WANTS_BRAILLE=$enableval], [WANTS_BRAILLE=yes])
+
+AS_IF([test "x$WANTS_BRAILLE" = "xyes"], [
+  AC_DEFINE([ENABLE_BRAILLE], [1], [Define to enable IPv6])
+  USES_BRAILLE=yes
+])
```

Comment:
This needs a small correction in the help text.

Reply:
fixing.